src/libpriv/rpmostree-postprocess.c

```diff
@@ -50,7 +50,6 @@
 #include "rpmostree-rust.h"
 
 typedef enum {
-  RPMOSTREE_POSTPROCESS_BOOT_LOCATION_BOTH,
   RPMOSTREE_POSTPROCESS_BOOT_LOCATION_NEW
 } RpmOstreePostprocessBootLocation;
```

Comment:
We can just drop this enum completely now, right?

Reply:
Yes but I have a followup PR to add `boot-location: modules` :smile:

Reply:
Ahh gotcha! Maybe just a one-liner in the commit msg this is prep for a new variant?